extensions/2.0/Khronos/KHR_materials_dispersion/README.md

```diff
@@ -0,0 +1,218 @@
+# KHR\_materials\_dispersion
+
+## Contributors
+
+- Ben Houston, Threekit [@bhouston](https://github.com/bhouston)
+- Bastian Sdorra, Dassault Systemes [@bsdorra](https://github.com/bsdorra)
+- Tobias Haeussler [@proog128](https://github.com/proog128)
+- Mike Bond, Adobe [@miibond](https://github.com/MiiBond)
+- Alexey Knyazev, Individual Contributor, [@lexaknyazev](https://github.com/lexaknyazev)
+- Eric Chadwick, DGG, [@echadwick-artist](https://github.com/echadwick-artist)
+- Alex Wood, AGI [@abwood](https://github.com/abwood)
+- Ed Mackey, AGI [@emackey](https://github.com/emackey)
+
+Copyright 2024 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Release Candidate
+
+## Dependencies
+
+- Written against the glTF 2.0 spec.
+- The `KHR_materials_volume` extension as this builds upon its volumetric effect.
+
+## Exclusions
+
+- This extension must not be used on a material that also uses `KHR_materials_pbrSpecularGlossiness`.
+- This extension must not be used on a material that also uses `KHR_materials_unlit`.
+
+## Overview
+
+This extension adds one parameter to the metallic-roughness material: `dispersion`.
+
+`dispersion` enables configuring the strength of the angular separation of colors (chromatic aberration) transmitting through a relatively clear volume.  It is an enhancement to the default ```KHR_materials_volume``` transmission model which assumes no dispersion.
+
+Optical dispersion is represented in terms of the Abbe number parameterization \( $V_d$ \).  The dispersion effect is a result of the wavelength-dependent index of refraction of a material.  Dispersion is a widely adopted parameter in modern PBR models.  It is present in both OpenPBR (as *transmission_dispersion_abbe_number*) and the Dassault Enterprise PBR Shading Model (as $V_d$).
+
+With the simplifying assumption that the index of refraction variance is linear,
+the Abbe number \( $V_d$ \) for visible light is computed as:
+$$V_d = \frac{n_d - 1}{n_F - n_C}$$
+
+where $n_F$, $n_d$ and $n_C$ are the indices of refraction at the wavelengths of the Fraunhofer F, d, and C spectral lines: 
+$$\lambda_F = 486.13 nm$$
+$$\lambda_d = 587.56 nm$$
+$$\lambda_C = 656.27 nm$$
+
+
+To calculate the index of refraction at a specific wavelength \( $\lambda$ \), we use the first two terms of Cauchy's equation:
+
+$$
+n(\lambda) = A + \frac{B}{\lambda^2}
+$$
+
+Given an Abbe number \( $V_d$ \) and the central index of refraction as specified by the ```KHR_materials_ior``` extension (assumed to be at the central wavelength, \( $n_d$ \)), the coefficients are defined as:
```

Comment:
`\( $n_d$ \))` -> `\( $n_d$ \)`

Reply:
Why? The closing parenthesis is needed to balance `(assumed`.

Reply:
Is there any easy way to avoid nested parenthesis here?  Or should we just leave it?

Reply:
We could just remove all parentheses around in-text variables. It's an editorial choice.

Reply:
Right, sorry. Looked awkward at first, but it's correct.